tests/utils_test.go

```diff
@@ -1,127 +0,0 @@
-package tests
```

Comment:
I think you deleted this file by accident

Reply:
Whoops, that explains the CI errors 😅  - I've restored the file